e2e/etcdctlv3_test.go

```diff
@@ -379,35 +387,45 @@ func ctlV3Del(cx ctlCtx, key string, num int) error {
 	return spawnWithExpects(cmdArgs, fmt.Sprintf("%d", num))
 }
 
-func ctlV3Watch(cx ctlCtx, key, value string) error {
-	watchCmd := []string{"watch", key}
-	if cx.watchRevision > 0 {
-		watchCmd = append(watchCmd, "--rev", strconv.Itoa(cx.watchRevision))
-	}
-
-	cmdArgs := ctlV3PrefixArgs(cx.epc, cx.dialTimeout)
+func ctlV3Watch(cx ctlCtx, key string, kvs ...kv) error {
+	watchCmd := append(ctlV3PrefixArgs(cx.epc, cx.dialTimeout), "watch")
 	if cx.interactive {
-		cmdArgs = append(cmdArgs, "watch", "--interactive")
+		watchCmd = append(watchCmd, "--interactive")
 	} else {
```

Comment:
@heyitsanthony https://github.com/coreos/etcd/blob/master/etcdctl/ctlv3/command/watch_command.go#L97-L102

watch command has its own parsing calls in interactive mode, so I had to generate the
command separately.

PTAL.

Thanks.